src/pipeline/wizard.ts

```diff
@@ -0,0 +1,182 @@
+              .then(() => `Pipeline '${inputStartPipeline.name}' successfully started`)
+              .catch((error) => Promise.reject(`Failed to start Pipeline with error '${error}'`))
+          );
+      }
+    }));
+
+
+    this.updateFunc();
+  }
+
+  dispose(): void {
+    if (this.disposed) {
+      return;
+    }
+
+    this.onDisposeEmitter.fire();
+    this.onDisposeEmitter.dispose();
+
+    this.editor.dispose();
+    super.dispose();
+  }
+
+  reveal(viewColumn: vscode.ViewColumn): void {
+    this.editor.reveal(viewColumn);
+  }
+
+  update(): void {
+    // TODO: 
+
+  }
+
+
+  private async doUpdate(): Promise<void> {
+    if (this.disposed) {
+      return;
+    }
+    const html = this.getHmlContent();
+    this.setContent(html);
+
+    try {
+      this.postMessage({ type: 'trigger', data: this.input.trigger });
+    } catch (err) {
+      console.error(err);
+    }
+  }
+
+  private postMessage(msg: {}): void {
+    if (!this.disposed) {
+      this.editor.webview.postMessage(msg);
+    }
+  }
+
+  private setContent(html: string): void {
+    this.editor.title = 'Start Pipeline';
+    // this.editor.iconPath = this.iconPath; //TODO: implement
+    this.editor.webview.options = getWebviewOptions();
+    this.editor.webview.html = html;
+  }
+
+  private getHmlContent(): string {
+    const nonce = new Date().getTime() + '' + new Date().getMilliseconds();
+    const rule = this.editor.webview.cspSource;
+    return `
+    <!DOCTYPE html>
+      <html lang="en" class="pf-m-redhat-font">
+        <head>
+          <meta http-equiv="Content-Security-Policy" content="default-src 'none'; img-src 'self' ${rule} http: https: data:; media-src 'self' ${rule} http: https: data:; script-src 'nonce-${nonce}'; style-src 'self' ${rule} 'unsafe-inline' http: https: data:; font-src 'self' ${rule} http: https: data:;">
+          <meta charset="utf-8" />
+          <meta name="viewport" content="width=device-width, initial-scale=1" />
+          <style type="text/css" media="screen, print">
+            @font-face {
+              font-family: "codicon";
+              src: url("${this.getFontPath()}") format("truetype");
+            }
+          </style>
+        </head>
+      <body>
+        <div id="root" style="width: 100%; height: 100%;"></div>
+        ${this.getScripts(nonce)}
+      </body>
+      </html>`;
+  }
+
+  private getScripts(nonce: string): string {
+    const out: string[] = [];
+    out.push(`<script
+				src="${escapeAttribute(this.editor.webview.asWebviewUri(vscode.Uri.file(path.join(contextGlobalState.extensionPath, '/out/webview/pipeline/index.js'))))}"
+				nonce="${nonce}"
+				charset="UTF-8"></script>`);
+    return out.join('\n');
+  }
+
+  private getImagesUri(): { [key: string]: string } {
+    const result: { [key: string]: string } = Object.create(null);
+    result['task'] = this.editor.webview.asWebviewUri(vscode.Uri.file(path.join(contextGlobalState.extensionPath, 'images', 'T.svg'))).toString();
```

Comment:
Do we use that?

Reply:
I am not using this code I will remove this code.